src/OpenTelemetry.Extensions.PersistentStorage.Api/OpenTelemetry.Extensions.PersistentStorage.Api.csproj

```
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

I think we're dumping net461 as it's end of life very soon now.

```suggestion
    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```